BrainPortal/lib/boutiques_file_name_verifier.rb

```diff
@@ -0,0 +1,86 @@
+
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2024
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# This module adds automatic verification of the
+# files (or directories ) names generated as output, which comply with CBRAIN restrictions.
+#
+# For exmaple, in the "inputs" section:
+#
+#   {
+#     "description": "The name of the folder to store outputs of XCPD processing.",
+#     "id": "output_dir",
+#     "name": "output_dir",
+#     "optional": false,
+#     "type": "String",
+#     "value-key": "[OUTPUT_DIR]",
+#     "default-value": "xcpd_output"
+#   }
+#
+# and in the "output-files" section:
```

Comment:
Remove lines 38-47 in the comment block. We do not need to talk about the output-files section within the module's documentation, we are only validating String inputs to be properly formatted like CBRAIN file names.